### PR TITLE
Fix Tag Tracking+ links

### LINKED ISSUE #694: Tag Tracking+ takes the person using it to tagged/tag/recent instead of just tagged.
xkit issues2

### PR DESCRIPTION
**Pull request by hobinjk**
Use `/tagged/word-word` instead of `/tagged/word+word/recent`.
Fixes #694

### CODE REVIEW

File: Extensions/classic_tags.js [hunk 1]
```diff
@@ -67,12 +67,21 @@ XKit.extensions.classic_tags = new Object({
 			$(".result_link").each(function() { $(this).attr("target", ""); });
 		}
 		if (XKit.extensions.classic_tags.preferences.redirect_to_tagged.value) {
-			$(".result_link").each(function() { $(this).attr("href", $(this).attr("href").replace("/search/", "/tagged/")); });
-		} else {
-			$(".result_link").each(function() { $(this).attr("href", $(this).attr("href").replace("/tagged/", "/search/")); });
+			$(".result_link").each(this.redirect_to_tagged);
```
Comment (Wolvan) [inline Extensions/classic_tags.js]: This needs to be ``XKit.extensions.classic_tags.redirect_to_tagged`` instead of ``this.redirect_to_tagged``

**Comment by AuraSweet** [comment]
I'm still confused, how does that help? When you click on the tag it goes straight to tagged/word/recent

File: Extensions/classic_tags.js [hunk 2]
```diff
@@ -209,9 +218,7 @@ XKit.extensions.classic_tags = new Object({
...
```
Comment (Wolvan) [inline Extensions/classic_tags.js]: Same as above

**Comment by AuraSweet** [comment]
I don't understand coding, none of that makes any sense to me.

File: Extensions/classic_tags.js (hunk 1 above)
Comment (hobinjk) [inline Extensions/classic_tags.js]: The way the function invocation works in this case `this` is equivalent to `XKit.extensions.classic_tags`, but I can change it for clarity's sake

Comment (Wolvan) [inline Extensions/classic_tags.js]: Not exactly, I get a ``Cannot read property 'call' of undefined``. Debugging the function shows me that this.redirect_to_tagged is indeed undefined

**Comment by Wolvan** [comment]
Confirmed to work now, @homu r+

**Pull request merged**